tests/test_visitors/test_ast/test_keywords/test_raise.py

```diff
@@ -130,3 +131,60 @@ def test_bare_raise(
     visitor.run()
 
     assert_errors(visitor, [])
+
+
+def test_raise_except(
+    assert_errors,
+    parse_ast_tree,
+    default_options,
+):
+    """Testing that bare `raise` is not allowed without an except block."""
+    code = """
+    def hello():
+        try:
+            x = 1
+        except Exception:
+            raise ValueError('1')
+    """
```

Comment:
Let's parametrize this test to test at least 2 cases: 

+ `raise SomeException()` inside `except` block
+ bare `raise` inside `except` block

```suggestion

@pytest.mark.parametrize('raise_statement', [
    "raise ValueError('1')",
    'raise',
])
def test_raise_inside_except(
    assert_errors,
    parse_ast_tree,
    default_options,
    raise_statement,
):
    """Ensure ``raise`` statement inside ``except`` block allowed."""
    code = """
        try:
            x = 1
        except Exception:
            {raise_statement}
    """.format(raise_statement=raise_statement)
```

Reply:
You can also create one parametrized test like suggested by Nikita in https://github.com/wemake-services/wemake-python-styleguide/pull/1998#discussion_r619993569 or me in https://github.com/wemake-services/wemake-python-styleguide/pull/1998#discussion_r619823903